### PR TITLE
fix: HTML addition for youtube link for pro platform #8407

### PR DESCRIPTION
**Pull request by aadak99**
Added link to nav menu as "Youtube tutorials"

I was having difficulty with setting up the pro dev container. I could see the (pro) in my git bash, but only po was produced not po_pro like expected. Regardless, I found the html page corresponding to the pro platform and put in a _blank link to youtube tutorial playlist. 

- Fixes: #8408

### CODE REVIEW

**Comment by alexgarel** [comment]
@aadak99, I fixed the description, as you put `fixes #[8407]`, the `[]` disabled the linking. Also it was #8408 (not your fault, there was a misleading #8407 in the title of the issue…)

**Review by alexgarel — CHANGES_REQUESTED** [review]
Thanks a lot for contributing @aadak99, though we have to handle translations. See my comment above.

File: templates/web/common/site_layout.tt.html [hunk 1]
```diff
@@ -63,6 +63,7 @@
 									<li class="divider"></li>
 
 									<li><label>[% lang("resources") %]</label></li>
+                  <li><a href="https://www.youtube.com/playlist?list=PLjAH-USadsF3xgSVfUUBS4we3XBr-1r55" target="_blank">Video tutorials</a></li>
```
[the remark this answers is not in the document]
Comment (aadak99) [reply templates/web/common/site_layout.tt.html:66]: thanks so much, I look forward to being a regular contributor. I still need to learn the various different tools and methods y'all use. Going forward would I be allowed to input the crowdin strings, I did one translation so far for a link in the wrong language. Wondering how I would go about inputting a new string.

**Review by aadak99 — COMMENTED** [review]
#8332

Added Producers link to top nav, with translatable string

**Pull request closed without merging**